lib/rubocop/cop/naming/predicate_name.rb

```diff
@@ -100,6 +128,7 @@ def on_def(node)
             method_name = node.method_name.to_s
 
             next if allowed_method_name?(method_name, prefix)
+            next if sorbet? && !boolean_sorbet_sig?(node)
```

Comment:
What if a codebase uses sorbet partially? They might want to enable this option, but in that case, any files/`def`s that have not been marked with `sig` will not fall back to the default behaviour. I think we should probably use the existing behaviour if `UseSorbetSigs` is enabled, but the method has no `sig`.